.changeset/fine-grapes-itch.md

```diff
@@ -0,0 +1,24 @@
+---
+"@fluidframework/datastore": minor
+"@fluidframework/datastore-definitions": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+"__section": feature
+---
+`minVersionForCollab` is now available on `IFluidDataStoreContext`
+
+`minVersionForCollab` is now passed down from the `ContainerRuntime` to the Datastore layer where it is made available for
+`SharedObject` construction.
+DDSes may optionally consume this value and use it to determine which sets of feature flags should be enabled.
+
+#### Interface changes
+
+All implementers of `IFluidDataStoreContext` should forward the `minVersionForCollab`
+value. Constraints for the interfaces that now provide the value via accessors or fields:
+
+- **@fluidframework/runtime-definitions: `IFluidDataStoreContext`** - Implementation (`FluidDataStoreContext`) must read
```

Comment:
Not sure I understand this. Any implementation of `IFluidDataStoreContext` must now implement this new property right?

Reply:
I'm also not sure that we need this bit. If there was a behavior change in `FluidDataStoreContext` or its concrete implementations (which are all `@internal`) we did it in this PR, right? We could document what happened if it somehow impacts consumers, but right now I read this like "this class now needs to do something in response to this change" as if it was an external consumer.